components/match2/wikis/rocketleague/match_group_input_custom.lua

```diff
@@ -207,6 +212,42 @@ function matchFunctions.getExtraData(match)
 	return match
 end
 
+function p._getCasterInformation(name, flag, id)
+	if String.isEmpty(flag) then
+		flag = Variables.varDefault(name .. '_flag')
+	end
+	if String.isEmpty(id) then
+		id = Variables.varDefault(name .. '_dn')
+	end
+	if String.isEmpty(flag) or String.isEmpty(id) then
+		local parent = Variables.varDefault('tournament_parent')
+		local pageName = mw.ext.TeamLiquidIntegration.resolve_redirect(name)
+		local data = mw.ext.LiquipediaDB.lpdb('broadcasters', {
+			conditions = '[[page::' .. pageName .. ']] AND [[parent::' .. parent .. ']]',
+			query = 'flag, id',
+			limit = 1,
+		})
```

Comment:
let parent default to the pagename so we avoid nil concats in case the unset variable as well as excluding pages that do not belong to any parent
```suggestion
		local parent = Variables.varDefault(
			'tournament_parent',
			mw.title.getCurrentTitle().text
		)
		local pageName = mw.ext.TeamLiquidIntegration.resolve_redirect(name)
		local data = mw.ext.LiquipediaDB.lpdb('broadcasters', {
			conditions = '[[page::' .. pageName .. ']] AND [[parent::' .. parent .. ']]',
			query = 'flag, id',
			limit = 1,
		})
```